### ISSUE TITLE
 The principle behind this project 

### ISSUE THREAD

**Issue by northtiger** [EddiePi/spark-DynamicMemory#1]
Hi,my project is also about the Spark memory optimization.I want to ask, what is the principle behind this project ?Can you overview? :dancing_men:  @EddiePi

**Comment by EddiePi** [comment]
@northtiger Hi. At first I intended to dynamically adjust the memory of executors at runtime. I modified some of the code in `UnifiedMemoryManager`. But the result was not as good. This project has been abandoned for a long time.

**Comment by northtiger** [comment]
Yeah,I know UnifiedMemoryManager, and I also want to modify this class. I think Spark cache is basis on LRU. But I do not think this is a perfect strategy. Let me try to do some work on it. 😄
